### PR TITLE
Update core-committers.md

### PR DESCRIPTION
**Pull request by comharris**
Added `Corey Robinson`

FYI, updated https://developers.mattermost.com/internal/rd-teams/ as well

### CODE REVIEW

**Comment by hanzei** [comment]
cc @corey-robinson

**Comment by DSchalla** [comment]
Thanks @comharris! Since the security team is not a traditional Dev Team we might should move it to a separate entry, similar to the build engineering team - Thoughts? Happy to do that in a separate PR of course.

**Comment by comharris** [comment]
@DSchalla I think that's a great idea!

Reckon it should be called something like "Security Engineers"? 0/5

LMK:

1. Title of the section
2. Location/position of the section (e.g. immediately below / above the "Build Engineers" section - or wherever)

Thanks!

**Comment by comharris** [comment]
@DSchalla Please see above when you have a chance. TY

**Comment by DSchalla** [comment]
Sorry @comharris! Yes, below Build Engineers is totally fine and the proposed title sounds good :+1:

File: site/content/contribute/getting-started/core-committers.md [hunk 1]
```diff
@@ -241,6 +236,19 @@ Build Engineers
     - @elisabeth.kulzer on [community.mattermost.com](https://community.mattermost.com/core/messages/@elisabeth.kulzer) and [@metanerd](https://github.com/metanerd) on GitHub
 - **<a name="jason.deland">Jason Deland</a>**
     - @jason.deland on [community.mattermost.com](https://community.mattermost.com/core/messages/@jason.deland) and [@jaydeland](https://github.com/jaydeland) on GitHub
+    
+Security Engineers
```
Comment (comharris) [inline site/content/contribute/getting-started/core-committers.md]: Thanks @DSchalla. Feel free to review if you'd like

**Comment by comharris** [comment]
Added Ibrahim; removed Claude. Also updated [R&D teams](https://github.com/mattermost/mattermost-developer-documentation/pull/531)

**Pull request merged**